src/Resources/config/twig.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sonata.admin_doctrine_orm.twig.extension"
+                 class="Sonata\DoctrineORMAdminBundle\Twig\Extension\SonataDoctrineORMAdminExtension"
+                 public="true"
```

Comment:
Why explicit public?

Reply:
It was a copy paste, didn't even saw that one, but this extension will be moved to Admin Bundle and will fix it there.